src/YieldDisburser.sol

```diff
@@ -10,60 +10,115 @@ import {Bread} from "../lib/bread-token-v2/src/Bread.sol";
 error AlreadyClaimed();
 
 contract YieldDisburser is OwnableUpgradeable {
-    address[] public breadchainProjects;
-    address[] public queuedProjectsForAddition;
-    address[] public queuedProjectsForRemoval;
-    address[] public breadchainVoters;
+    // Storage of intristic , assumed constants
```

Comment:
constants should be capitalized

Reply:
well it's like they're near constants, not true constants , I don't know if there's a convention for that.